app/controllers/comments/votes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Comments::VotesController < ApplicationController
+  before_action :ensure_current_user
+  before_action :set_comment
+
+  def create
+    current_user.likes @comment
+    if @comment.parent_id
+      closest_comment_by_votes = @comment.siblings.min_by {|c| (c.cached_votes_up - @comment.cached_votes_up).abs }
+      if @comment.cached_votes_up >= closest_comment_by_votes.cached_votes_up
+        closest_comment_by_votes.prepend_sibling(@comment)
+      else
+        closest_comment_by_votes.append_sibling(@comment)
+      end
+    end
+
+    redirect_to :back
+  end
+
+  def destroy
+    current_user.unlike @comment
+    if @comment.parent_id
+      closest_comment_by_votes = @comment.siblings.min_by {|c| (c.cached_votes_up - @comment.cached_votes_up).abs }
+      if @comment.cached_votes_up >= closest_comment_by_votes.cached_votes_up
+        closest_comment_by_votes.prepend_sibling(@comment)
+      else
+        closest_comment_by_votes.append_sibling(@comment)
+      end
```

Comment:
Aren't these two blocks the same? You could capture this in a method so you have one definitive source of knowledge for prepending/appending comments based on votes.